[Info]

;Вариант. 0 - если у задачи нет вариантов. Числа 1, 2, ... если есть.
Variant=0

;Условие
Text="Что будет выведено на экран после выполнения фрагмента программы</p><p>int k = 1;</p><p>try</p><p>{</p><p>if (k == 0) throw new int(5);</p><p>if (k == 1) throw 5.;</p><p>if (k == 2) throw '5';</p><p>if (k == 3) throw 5;</p><p>}</p><p>catch (int*)</p><p>{ cout &lt;&lt; \"catch int*\" &lt;&lt; endl;}</p><p>catch (int)</p><p>{ cout &lt;&lt; \"catch int\" &lt;&lt; endl; }</p><p>catch (float)</p><p>{ cout &lt;&lt; \"catch float\" &lt;&lt; endl;}</p><p>catch (...)</p><p>{ cout &lt;&lt; \"catch \" &lt;&lt; endl; }</p><p>"

;Сложность в диапазоне [1-5]
Difficulty=2

;Тема
Topic="Перехват исключений"

;ID Темы
TopicId=581

;Автор
Author="Демидова А.Н."

;Имя решателя (не изменять)
Solver="Test"

[Solution]
Type = "single"

Single = "catch "
ResponseOptions[]="catch "
ResponseOptions[]="catch int*"
ResponseOptions[]="catch int"
ResponseOptions[]="catch float"



[DataBase]
TaskId=1054
GroupTaskId=1006
